mappings/net/minecraft/client/render/debug/CaveDebugRenderer.mapping

```diff
@@ -1 +1,11 @@
 CLASS net/minecraft/class_859 net/minecraft/client/render/debug/CaveDebugRenderer
+	FIELD field_4505 client Lnet/minecraft/class_310;
+	FIELD field_4506 caveCenters Ljava/util/List;
+	FIELD field_4507 nodeCenters Ljava/util/Map;
+	FIELD field_4508 nodeWeights Ljava/util/Map;
```

Comment:
A node is a fixed point along a connected path. They're like the things that come between "rungs".

You can think of it light the traffic signals along a busy highway, where every intersection is a node where the path can either turn or continue straight.

In this case the caves for the debug renderer is treated like a path, where each node is a position that is "part" of the cave system. I named `caveCenters` being the important parts of the caves, or the parts where the cave branches were initially spawned from.


`nodeWeights` is used when drawing each node to determine how large their boxes should be. It could probably be called `nodeSize` but in this case I went with `higher weight = larger size when displayed`.